cmd/health-agent/probers.go

```diff
@@ -15,6 +16,7 @@ import (
 
 func setupProbers() (*proberlist.ProberList, error) {
 	pl := proberlist.New("/probers")
+	go pl.CreateAndAdd(aws.Register, "/sys/aws", 0)
```

Comment:
Here is my straw man API proposal.

// Returned directory is hidden
func NewHiddenDirectory() *DirectorySpec {
}

// dirSpec becomes visible at path.
func UnhideDirectory(path string, dirSpec *DirectorySpec) error {
}